gradio/routes.py

```diff
@@ -543,7 +559,7 @@ async def file(path_or_url: str, request: fastapi.Request):
 
             is_dir = abs_path.is_dir()
 
-            if in_blocklist or is_dir:
+            if is_dir or in_blocklist:
```

Comment:
curious why :)

Reply:
Typo hehe. Originally moved `is_blocklist` to be a separate check but realized we don't have to do that